faunadb/query.py

```diff
@@ -19,32 +18,24 @@
 
 def let(vars, in_expr):
   """See the `docs <https://faunadb.com/documentation/queries#basic_forms>`__."""
-  return {"let": vars, "in": in_expr}
+  # vars is a dict, but we don't want it converted to an object expression.
+  vars_expr = _Expr(_convert_values(vars))
+  return _q({"let": vars_expr, "in": in_expr})
 
 
 def var(var_name):
   """See the `docs <https://faunadb.com/documentation/queries#basic_forms>`__."""
-  return {"var": var_name}
+  return _q({"var": var_name})
 
 
 def if_expr(condition, then, else_):
   """See the `docs <https://faunadb.com/documentation/queries#basic_forms>`__."""
-  return {"if": condition, "then": then, "else": else_}
+  return _q({"if": condition, "then": then, "else": else_})
 
 
 def do(*expressions):
   """See the `docs <https://faunadb.com/documentation/queries#basic_forms>`__."""
-  return {"do": _varargs(expressions)}
-
-
-def object(**fields):
-  """See the `docs <https://faunadb.com/documentation/queries#basic_forms>`__."""
-  return {"object": fields}
-
-
-def quote(expr):
-  """See the `docs <https://faunadb.com/documentation/queries#basic_forms>`__."""
-  return {"quote": expr}
+  return _q({"do": _varargs(expressions)})
 
 
 def lambda_query(func):
```

Comment:
We need to document somewhere that lambdas with multiple arguments become destructuring lambdas. Where should we put this if lambda_query and to_query are both hidden? Maybe in the query.py module docstring?